samples/authorization/csm_authorization_proxy_server_v200-alpha.yaml

```diff
@@ -55,6 +55,9 @@ spec:
           authorizationControllerReplicas: 1
           leaderElection: true
 
+          # controllerReconcileInterval: interval for the authorization controllers to reconcile with Redis.
+          controllerReconcileInterval: 5m
```

Comment:
Can this be added to all the v2.0.0-alpha versioned CRs (in unit tests and E2E CRs)?